Sources/Journify/Utilities/JSON.swift

```diff
@@ -501,6 +515,37 @@ extension Dictionary where Key == String, Value == Any {
         
         return mapped
     }
+
+    func hashedValues() -> [String: Any] {
+        var hashedDict = [String: Any]()
+        
+        for (key, value) in self {
+            if let stringValue = value as? String {
+                // If the value is a String, hash it using SHA-256
+                let hashedValue = SHA256.hash(data: Data(stringValue.utf8)).compactMap { String(format: "%02x", $0) }.joined()
+                hashedDict[key] = hashedValue
+            } else if let intValue = value as? Int {
+                // If the value is an Int, convert it to String and then hash it using SHA-256
+                let stringValue = String(intValue)
+                let hashedValue = SHA256.hash(data: Data(stringValue.utf8)).compactMap { String(format: "%02x", $0) }.joined()
+                hashedDict[key] = hashedValue
+            } else if let floatValue = value as? Float {
+                // If the value is a Float, convert it to String and then hash it using SHA-256
+                let stringValue = String(floatValue)
+                let hashedValue = SHA256.hash(data: Data(stringValue.utf8)).compactMap { String(format: "%02x", $0) }.joined()
+                hashedDict[key] = hashedValue
+            } else if let boolValue = value as? Bool {
+                // If the value is a Bool, convert it to String and then hash it using SHA-256
+                let stringValue = String(boolValue)
+                let hashedValue = SHA256.hash(data: Data(stringValue.utf8)).compactMap { String(format: "%02x", $0) }.joined()
+                hashedDict[key] = hashedValue
+            } else {
+                // If the value is not a String, Int, Float, or Bool, leave it unchanged
+                hashedDict[key] = value
+            }
```

Comment:
Only strings need hashing, you can safely send the other traits without hash 🙏

Reply:
Sorry, but why don't we need to hash other formats?